projects/showcase/cypress/e2e/live-update-feature.cy.ts

```diff
@@ -19,6 +19,7 @@ describe('Live Update', () => {
       });
 
       it('Then I should see the map image change and the map center coordinates are different', () => {
+        driver.when.wait(1000);
```

Comment:
Simply add this to the `beforeEach` and change the name of the context to `When I wait three seconds.`